packages/dd-trace/src/config.js

```diff
@@ -674,6 +675,7 @@ class Config {
       DD_TRACE_HEADER_TAGS,
       DD_TRACE_LEGACY_BAGGAGE_ENABLED,
       DD_TRACE_MEMCACHED_COMMAND_ENABLED,
+      DD_TRACE_MIDDLEWARE_ENABLED,
```

Comment:
The environment variable doesn't match the name of the programmatic option.

Reply:
It also doesn't have a corresponding test.

Reply:
> The environment variable doesn't match the name of the programmatic option.

Please provide suggested values for both. @tlhunter @rochdev please coordinate on this because right now you folks have both made naming suggestions.

> It also doesn't have a corresponding test.

I don't understand this comment. In this same review, you commented on the corresponding test. What additional testing are you looking for?

Reply:
> The environment variable doesn't match the name of the programmatic option.

Please provide suggested values for both. @tlhunter @rochdev please coordinate on this because right now you folks have both made naming suggestions.

> It also doesn't have a corresponding test.

I don't understand this comment. In this same review, you commented on the corresponding test. What additional testing are you looking for?

Reply:
> Please provide suggested values for both.

I don't mind the name, but it should map between programmatic and env var.

> I don't understand this comment. In this same review, you commented on the corresponding test.

The plugin test yes, but there isn't a test for the option itself (config.spec.js)

Reply:
>I don't mind the name, but it should map between programmatic and env var.

So renaming the env var to `DD_TRACE_MIDDLEWARE_TRACING_ENABLED` would be sufficient? That's a lot of `TRACE` in the name, but I guess that's fine.

Reply:
> That's a lot of TRACE in the name, but I guess that's fine.

We did name the library `dd-trace` after all 😅